kornia/feature/responses.py

```diff
@@ -268,6 +299,25 @@ def forward(self, input: Tensor, sigmas: Optional[Tensor] = None) -> Tensor:
         return dog_response(input)
 
 
+class BlobDoGSingle(Module):
+    r"""Module that calculates Difference-of-Gaussians blobs.
+
+    See :func:`~kornia.feature.dog_response_single` for details.
+    """
+
+    def __init__(self, sigma1: float = 1.0, sigma2: float = 1.6) -> None:
+        super().__init__()
+        self.sigma1 = sigma1
+        self.sigma2 = sigma2
+        return
+
+    def __repr__(self) -> str:
+        return f'{self.__class__.__name__}, sigma1={self.sigma1}, sigma2={self.sigma2})'
+
+    def forward(self, input: Tensor, sigmas: Optional[Tensor] = None) -> Tensor:
```

Comment:
`sigmas` should do something here?

Reply:
Sigmas are here to insure compatibility with other ScaleSpaceDetector we have.